admin_guide/pruning_resources.adoc

```diff
@@ -185,6 +185,23 @@ streams. Images are non-namespaced resources. Therefore, limiting pruning to a
 particular namespace makes it impossible to calculate their current usage.
 ====
 
+It's important to know that by default the integrated registry caches blobs metadata
+to reduce the number of requests to the storage and increase the speed of processing
+the request. Pruning doesn't update the integrated registry cache. It means that images
+that will be pushed after pruning and that contain pruned layers will be broken.
```

Comment:
s/that will be pushed/that are pushed/

s/will be broken/will be broken because the pruned layers that have metadata in the cache will not be pushed, resulting in those layers being missing after the push is complete."